spec.yml

```diff
@@ -3607,14 +3652,91 @@ parameters:
     description: 'Number of results per page in paginated response'
     type: number
     format: int32
-  uuid:
-    name: uuid
+  refreshTokenID:
+    name: refreshTokenID
+    in: path
+    required: true
+    type: string
+  datasourceID:
```

Comment:
Is it possible to fill these in with a `ref` to a generic `uuidPathParam` for all the `in: path`, `required: true`, `type: string`, `format: uuid` pieces, and then just provide the name?

Reply:
@jisantuc lemme try

Reply:
yea, it does not seem like this is possible -- though it could be less repetitive if it was.